cli/flox/doc/flox-show.md

```diff
@@ -11,19 +11,17 @@ flox-show - show detailed information about a single package
 
 # SYNOPSIS
 
-flox [ `<general-options>` ] show `<name>` [ \--all ]
+```
+flox [ <general-options> ] show <search-term>
```

Comment:
Suggestion: replace search term with package id

Reply:
Search term is used in the help description so I'd prefer it be consistent